epictrack-api/migrations/versions/09a79ca8e8fd_add_none_ministry.py

```diff
@@ -0,0 +1,60 @@
+"""Add None ministry in ministries table
+
+Revision ID: 09a79ca8e8fd
+Revises: 65cb91595d6a
+Create Date: 2024-01-11 13:19:49.744798
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '09a79ca8e8fd'
+down_revision = '65cb91595d6a'
+branch_labels = None
+depends_on = None
+
+NoneMinistryName = 'None'
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    ministries = sa.table(
+        'ministries',
+        sa.Column('id', sa.Integer),
+        sa.Column('name', sa.String),
+        sa.column('abbreviation', sa.String),
+        sa.Column('is_active', sa.Boolean),
+        sa.Column('is_deleted', sa.Boolean),
+        sa.Column('sort_order', sa.Integer),
+    )
+
+    ## select greatest id from ministries
+    conn = op.get_bind()
+    result = conn.execute(
+        sa.text(
+            "SELECT max(id) FROM ministries"
+        )
+    )
+    new_id = result.fetchone()[0] + 1
+
+    op.execute(
+        ministries.insert().values(
+            id=new_id,

```

Comment:
We don't need to pass `id` to insert construct. It will fetch the next id value from database  by default.

Reply:
good point, thanks